test/tests/spiped-basics/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+[ "$DEBUG" ] && set -x
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+clientImage='buildpack-deps:buster-curl'
+# ensure the clientImage is ready and available
+if ! docker image inspect "$clientImage" &> /dev/null; then
+	docker pull "$clientImage" > /dev/null
+fi
+
+# The keygen below is a bit ugly, because spiped-generate-key.sh expects /spiped/key to be a directory (you can't bind mount a non-existing file),
+# but the entrypoint expects /spiped/key to be the actual keyfile.
```

Comment:
It seems like the extra complexity described here is because either `spiped-generate-key.sh` or `docker-entrypoint.sh` should be adjusted, but :shrug: that's your choice.

Reply:
Yeah, it should. But I don't want to break compatibility here and I'm not sure what the correct solution would be. I certainly don't want to expect the users bind mounting a full folder containing a single keyfile vs. just the keyfile.